tests/testthat/test-include-gld-purchases.R

```diff
@@ -0,0 +1,102 @@
+constants <- tibble::tibble(
+  pnr = "2132131231",
+  eksd = "20210101",
+  volume = 1.1,
+  apk = 1.1,
+  indo = "324314324",
+  name = "random",
+  vnr = "123123"
+)
+
+lmdb <- tibble::tribble(
+  ~atc,
+  "A10abc",
+  "A10",
+  "A10123",
+  "A11",
+  "A21",
+  "B10A10",
+) |>
+  dplyr::bind_cols(constants)
+
+expected <- tibble::tribble(
+  ~atc,
+  "A10abc",
+  "A10",
+  "A10123",
+) |>
+  dplyr::bind_cols(constants) |>
+  dplyr::rename(date = eksd) |>
+  dplyr::relocate(atc, .after = date)
+
+test_that("dataset needs expected variables", {
```

Comment:
The algorithm needs either the "name" or "vnr" variable to work, so we should add this to the tests and the inclusion function itself (e.g. check for any of the variables in the input data and use the one that is available (and use "name" by default if both are available).

Reply:
After testing, it turns out we don't need "name" nor "vnr" any more, so this comment is irrelevant now.